Views/Users/Display.cshtml

```diff
@@ -34,6 +34,7 @@
                 @switch (Model.AccountBadge)
```

Comment:
Maybe at this point we can change the switch statement so that we are actually using the value in the `Display` attribute? Can you look into this? Otherwise I will.